example-workflows/prep-release.yml

```diff
@@ -12,6 +12,10 @@ on:
       post_version_spec:
         description: "Post Version Specifier"
         required: false
+      silent:
```

Comment:
Wondering if that new option should be added to the example workflows?

Probably most repos will not use it. So the motivation for not adding it would be to avoid confusing people with a new input.
And the repos that need it can add the input to their workflows directly.

Reply:
Commenting it